project/BRANCHES-AND-TAGS.md

```diff
@@ -13,16 +13,19 @@ Keep in mind that release branches only accept bug and security fixes; new featu
 
 Currently (and previously) maintained release branches are documented in the table below:
 
-| Branch Name                       | Sponsoring Maintainer(s)                                | Contribution Status   | Expected End of Maintenance  | Known Distributors                  |
-|-----------------------------------|---------------------------------------------------------|-----------------------|------------------------------|-------------------------------------|
-| master (development branch)       | The Moby Project [MAINTAINERS](../MAINTAINERS)          | N/A                   | -                            | N/A                                 |
-| 27.x                              | The Moby Project [MAINTAINERS](../MAINTAINERS)          | Maintained            | After 28.x                   | [Docker, Inc.](https://docker.com)  |
-| 26.1                              | @cpuguy83                                               | Maintained (security) | TBD                          | [Microsoft](https://microsoft.com)  |
-| 25.0                              | @corhere                                                | Maintained            | TBD                          | [Amazon][al2023]                    |
-| 23.0                              | @corhere                                                | Maintained            | [2025-05-19][mcr23-maint]    | [Mirantis](https://mirantis.com)    |
+| Branch Name                       | Sponsoring Maintainer(s)                                | Contribution Status   | Expected End of Maintenance  | Known Distributors                       |
+|-----------------------------------|---------------------------------------------------------|-----------------------|------------------------------|------------------------------------------|
+| master (development branch)       | The Moby Project [MAINTAINERS](../MAINTAINERS)          | N/A                   | -                            | N/A                                      |
+| 27.x                              | The Moby Project [MAINTAINERS](../MAINTAINERS)          | Maintained            | After 28.x                   | [Docker, Inc.][docker],[Microsoft][msft] |
+| 26.1                              | @cpuguy83                                               | Maintained (security) | TBD                          | [Microsoft][msft]                        |
```

Comment:
@cpuguy83 not sure if we should keep the status for 26.1 as "maintained", or if this should become end of maintenance if no active maintenance is happening.

also cc @austinvazquez in case things are being decided on "what's the next version to package" for Amazon Linux.

Reply:
I would say not actively maintaining 26.x.

Reply:
Thanks! Let me update